h2o-r/tests/testdir_algos/generic/runit_generic_mojo_model_ids.R

```diff
@@ -16,36 +16,32 @@ test.model.generic.mojo.ids <- function() {
   mojo_original_path <- paste0(mojo_original_name)
     
   # Import MOJO
-  print("Import MOJO")
+  print("Import MOJO with a given model_id")
   mojo_model <- h2o.import_mojo(mojo_original_path, original_model_id)
```

Comment:
I think we should have some sort of force parameter in case that model_id is exists similar to 
`save_mojo(self, path="", force=False,`

Reply:
that is a good idea - we could do the same for regular models as well